src/classes/RD2_QueueableService.cls

```diff
@@ -212,4 +212,68 @@ public class RD2_QueueableService {
                 .cancelCommitments(commitmentRDs);
         }
     }
+
+    /**
+     * @description Queues up Opp name update service to recalculate Opportunity name.
+     * @param updatedOpps Updated opportunities
+     */
+    public static void enqueueOppNameUpdateService(List<Opportunity> updatedOpps) {
```

Comment:
Let's rename `enqueueOppNameUpdateService` to `enqueueOpportunityNaming` or `enqueueOppNamingService`.